browser.js

```diff
@@ -7,7 +7,9 @@ let main=async function() {
     initButton.innerText="Start";
     initButton.addEventListener("click", function() {
         let script=async function() {
-            let fileRefs=await window.showOpenFilePicker();
+            let fileRefs=await window.showOpenFilePicker({
+                multiple: true
+            });
```

Comment:
@webdev03 Here's the section I'm wondering about.

Reply:
Oh Ok

Reply:
> Oh Ok

Does that look right?